scripts/release/release-test.R

```diff
@@ -461,17 +469,36 @@ if(url.exists(armadillo_url)) {
   exit_test(msg)
 }
 
-cat("Location of molgenis-service-armadillo on your PC (for default ~/git/, press enter, if not available press x): ")
-service_location <- readLines("stdin", n=1)
+service_location = Sys.getenv("GIT_CHECKOUT_DIR")
+if(service_location == ""){
+  cat("Location of molgenis-service-armadillo on your PC (for default ~/git/, press enter, if not available press x): ")
+  service_location <- readLines("stdin", n=1)
+} else {
+   cat("GIT_CHECKOUT_DIR from '.env' file\n")
+}
+
 dest <- ""
 
-cat("Enter password for admin user (basic auth)\n")
-admin_pwd<- getPass::getPass()
+admin_pwd = Sys.getenv("ADMIN_PASSWORD")
+if(service_location == ""){
+  cat("Enter password for admin user (basic auth)\n")
+  admin_pwd<- getPass::getPass()
+} else {
+  cat("ADMIN_PASSWORD from '.env' file\n")
+}
 
-cat("Enter your OIDC username (email): ")
-user <- readLines("stdin", n=1)
+user = Sys.getenv("OIDC_EMAIL")
+if(user == ""){
+  cat("Enter your OIDC username (email): ")
+  user <- readLines("stdin", n=1)
+} else {
+   cat("OIDC_EMAIL from '.env' file\n")
+}
 
 if(service_location == "") {
+  # FIXME: as we are in same git repo path could be relative to this path?
+  # ../../data/shared-lifecycle/
+  # guess we have to download/copy to a tmp directory before uploading to armadillo.
```

Comment:
Completely agree, use the `.parquet` test files by default stored in `../../data/shared-lifecycle`